aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -215,6 +216,20 @@ module aptos_framework::aptos_governance {
         execution_hash: vector<u8>,
         metadata_location: vector<u8>,
         metadata_hash: vector<u8>,
+    ) acquires GovernanceConfig, GovernanceEvents {
+        create_single_step_or_multi_step_proposal(proposer, stake_pool, execution_hash, metadata_location, metadata_hash, false);
+    }
+
+    /// Create a single-step or multi-step proposal with the backing `stake_pool`.
+    /// @param execution_hash Required. This is the hash of the resolution script. When the proposal is resolved,
+    /// only the exact script with matching hash can be successfully executed.
+    public entry fun create_single_step_or_multi_step_proposal(
```

Comment:
nit: create_single_step_or_multi_step_proposal is a bit long. What do you think about create_proposal_v2? Or some other better name.